mmcv/datasets/pipelines/transforms.py

```diff
@@ -32,7 +33,10 @@ class Normalize:
         >>> results = normalize(results)
     """
 
-    def __init__(self, mean: Sequence, std: Sequence, to_rgb: bool = True):
+    def __init__(self,
+                 mean: Union[int, Sequence],
+                 std: Union[int, Sequence],
```

Comment:
Not only `int` but also `float`.
```suggestion
from numbers import Number
...
    def __init__(self,
                 mean: Union[Number, Sequence],
                 std: Union[Number, Sequence],
```